script/Ciel/Day 9/09_01_CIEL09_4.txt

```diff
@@ -642,7 +642,7 @@ She's a butcher that shows no concern about other people's lives, who's neverthe
 -- それが俺に聞かせたかった本題らしい。
 That seems to be what she wanted me to hear the most.
 }
-[sha:670420aaa3cb10cfc520fd1f52aebab89524bae4]{
+[offset:23232]{
```

Comment:
Why are all of these now offsets if they're still the same?

Reply:
Thinking about it, I added the offsets on the wrong files.